web/gui/dashboard_info.js

```diff
@@ -609,6 +609,12 @@ netdataDashboard.menu = {
         info: undefined
     },
 
+    'netframework': {
+        title: '.NET Framework',
+        icon: '<i class="fas fa-laptop-code"></i>',
```

Comment:
@thiagoftsm please open a ticket for @christophidesp if you need help finding the right icon. But I think we should always try and get the right icon for enterprise software.

Reply:
Done, I made the issue https://github.com/netdata/netdata/issues/14448.